contracts/getters.gno

```diff
@@ -0,0 +1,37 @@
+package gnoswap_optimizer
+
+func GetVault(tokenId uint64) Vault {
```

Comment:
Most of getters in `gnoswap` is meant be call from off-chain (abci_query or adena or gnokey).

If `GetVault()` is same purpose, you should return primitive type.

Reply:
I thought this function would be more used other realms and the other getters more for rpc usage.

I just didn't split into two files as I don't have a lot of getters